### DISCUSSION TITLE
文字检测模型，模型不训练，也不报错。

### DISCUSSION THREAD

**Discussion by kikiyaYang** [PaddlePaddle/PaddleOCR#12299] [Q&A] [answered]
以下是我的训练过程，他卡在'ppocr INFO: During the training process, after the 0th iteration, an evaluation is run every 20 iterations'，不再训练。
$ python3 tools/train.py -c configs/det/det_r18_vd_ct_test.yml -o Optimizer.base_lr=0.0001
[2024/03/29 16:06:06] ppocr INFO: Architecture :
[2024/03/29 16:06:06] ppocr INFO:     Backbone :
[2024/03/29 16:06:06] ppocr INFO:         layers : 18
[2024/03/29 16:06:06] ppocr INFO:         name : ResNet_vd
[2024/03/29 16:06:06] ppocr INFO:     Head :
[2024/03/29 16:06:06] ppocr INFO:         hidden_dim : 128
[2024/03/29 16:06:06] ppocr INFO:         in_channels : 512
[2024/03/29 16:06:06] ppocr INFO:         name : CT_Head
[2024/03/29 16:06:06] ppocr INFO:         num_classes : 3
[2024/03/29 16:06:06] ppocr INFO:     Neck :
[2024/03/29 16:06:06] ppocr INFO:         name : CTFPN
[2024/03/29 16:06:06] ppocr INFO:     Transform : None
[2024/03/29 16:06:06] ppocr INFO:     algorithm : CT
[2024/03/29 16:06:06] ppocr INFO:     model_type : det
[2024/03/29 16:06:06] ppocr INFO: Eval :
[2024/03/29 16:06:06] ppocr INFO:     dataset :
[2024/03/29 16:06:06] ppocr INFO:         data_dir : ./train_data/train/
[2024/03/29 16:06:06] ppocr INFO:         label_file_list : ['./train_data/xml2train_label.txt']
[2024/03/29 16:06:06] ppocr INFO:         name : SimpleDataSet
[2024/03/29 16:06:06] ppocr INFO:         ratio_list : [1.0]
[2024/03/29 16:06:06] ppocr INFO:         transforms :
[2024/03/29 16:06:06] ppocr INFO:             DecodeImage :
[2024/03/29 16:06:06] ppocr INFO:                 channel_first : False
[2024/03/29 16:06:06] ppocr INFO:                 img_mode : RGB
[2024/03/29 16:06:06] ppocr INFO:             CTLabelEncode : None
[2024/03/29 16:06:06] ppocr INFO:             ScaleAlignedShort : None
[2024/03/29 16:06:06] ppocr INFO:             NormalizeImage :
[2024/03/29 16:06:06] ppocr INFO:                 order : hwc
[2024/03/29 16:06:06] ppocr INFO:             ToCHWImage : None
[2024/03/29 16:06:06] ppocr INFO:             KeepKeys :
[2024/03/29 16:06:06] ppocr INFO:                 keep_keys : ['image', 'shape', 'polys', 'texts']
[2024/03/29 16:06:06] ppocr INFO:     loader :
[2024/03/29 16:06:06] ppocr INFO:         batch_size_per_card : 1
[2024/03/29 16:06:06] ppocr INFO:         drop_last : False
[2024/03/29 16:06:06] ppocr INFO:         num_workers : 2
[2024/03/29 16:06:06] ppocr INFO:         shuffle : False
[2024/03/29 16:06:06] ppocr INFO: Global :
[2024/03/29 16:06:06] ppocr INFO:     cal_metric_during_train : False
[2024/03/29 16:06:06] ppocr INFO:     checkpoints : None
[2024/03/29 16:06:06] ppocr INFO:     distributed : False
[2024/03/29 16:06:06] ppocr INFO:     epoch_num : 600
[2024/03/29 16:06:06] ppocr INFO:     eval_batch_step : [0, 20]
[2024/03/29 16:06:06] ppocr INFO:     infer_img : doc/imgs/test.jpg
[2024/03/29 16:06:06] ppocr INFO:     log_smooth_window : 20
[2024/03/29 16:06:06] ppocr INFO:     pretrained_model : ./pretrain_models/ResNet18_vd_pretrained.pdparams
[2024/03/29 16:06:06] ppocr INFO:     print_batch_step : 10
[2024/03/29 16:06:06] ppocr INFO:     save_epoch_step : 10
[2024/03/29 16:06:06] ppocr INFO:     save_inference_dir : inference/test
[2024/03/29 16:06:06] ppocr INFO:     save_model_dir : ./output/det_ct/
[2024/03/29 16:06:06] ppocr INFO:     save_res_path : ./output/det_ct/predicts_ct.txt
[2024/03/29 16:06:06] ppocr INFO:     use_gpu : True
[2024/03/29 16:06:06] ppocr INFO:     use_visualdl : False
[2024/03/29 16:06:06] ppocr INFO: Loss :
[2024/03/29 16:06:06] ppocr INFO:     name : CTLoss
[2024/03/29 16:06:06] ppocr INFO: Metric :
[2024/03/29 16:06:06] ppocr INFO:     main_indicator : f_score
[2024/03/29 16:06:06] ppocr INFO:     name : CTMetric
[2024/03/29 16:06:06] ppocr INFO: Optimizer :
[2024/03/29 16:06:06] ppocr INFO:     base_lr : 0.0001
[2024/03/29 16:06:06] ppocr INFO:     lr :
[2024/03/29 16:06:06] ppocr INFO:         end_lr : 0.0
[2024/03/29 16:06:06] ppocr INFO:         epochs : 600
[2024/03/29 16:06:06] ppocr INFO:         learning_rate : 0.001
[2024/03/29 16:06:06] ppocr INFO:         name : Linear
[2024/03/29 16:06:06] ppocr INFO:         power : 0.9
[2024/03/29 16:06:06] ppocr INFO:         step_each_epoch : 10
[2024/03/29 16:06:06] ppocr INFO:     name : Adam
[2024/03/29 16:06:06] ppocr INFO: PostProcess :
[2024/03/29 16:06:06] ppocr INFO:     box_type : poly
[2024/03/29 16:06:06] ppocr INFO:     name : CTPostProcess
[2024/03/29 16:06:06] ppocr INFO: Train :
[2024/03/29 16:06:06] ppocr INFO:     dataset :
[2024/03/29 16:06:06] ppocr INFO:         data_dir : ./train_data/train/
[2024/03/29 16:06:06] ppocr INFO:         label_file_list : ['./train_data/xml2train_label.txt']
[2024/03/29 16:06:06] ppocr INFO:         name : SimpleDataSet
[2024/03/29 16:06:06] ppocr INFO:         ratio_list : [1.0]
[2024/03/29 16:06:06] ppocr INFO:         transforms :
[2024/03/29 16:06:06] ppocr INFO:             DecodeImage :
[2024/03/29 16:06:06] ppocr INFO:                 channel_first : False
[2024/03/29 16:06:06] ppocr INFO:                 img_mode : RGB
[2024/03/29 16:06:06] ppocr INFO:             CTLabelEncode : None
[2024/03/29 16:06:06] ppocr INFO:             RandomScale : None
[2024/03/29 16:06:06] ppocr INFO:             MakeShrink : None
[2024/03/29 16:06:06] ppocr INFO:             GroupRandomHorizontalFlip : None
[2024/03/29 16:06:06] ppocr INFO:             GroupRandomRotate : None
[2024/03/29 16:06:06] ppocr INFO:             GroupRandomCropPadding : None
[2024/03/29 16:06:06] ppocr INFO:             MakeCentripetalShift : None
[2024/03/29 16:06:06] ppocr INFO:             ColorJitter :
[2024/03/29 16:06:06] ppocr INFO:                 brightness : 0.125
[2024/03/29 16:06:06] ppocr INFO:                 saturation : 0.5
[2024/03/29 16:06:06] ppocr INFO:             ToCHWImage : None
[2024/03/29 16:06:06] ppocr INFO:             NormalizeImage : None
[2024/03/29 16:06:06] ppocr INFO:             KeepKeys :
[2024/03/29 16:06:06] ppocr INFO:                 keep_keys : ['image', 'gt_kernel', 'training_mask', 'gt_instance', 'gt_kernel_instance', 'training_mask_distance', 'gt_distance']
[2024/03/29 16:06:06] ppocr INFO:     loader :
[2024/03/29 16:06:06] ppocr INFO:         batch_size_per_card : 1
[2024/03/29 16:06:06] ppocr INFO:         drop_last : True
[2024/03/29 16:06:06] ppocr INFO:         num_workers : 2
[2024/03/29 16:06:06] ppocr INFO:         shuffle : True
[2024/03/29 16:06:06] ppocr INFO: profiler_options : None
[2024/03/29 16:06:06] ppocr INFO: train with paddle 2.4.2 and device Place(gpu:0)
[2024/03/29 16:06:06] ppocr INFO: Initialize indexs of datasets:['./train_data/xml2train_label.txt']
list index out of range
[2024/03/29 16:06:06] ppocr INFO: Initialize indexs of datasets:['./train_data/xml2train_label.txt']
W0329 16:06:06.432868 18376 gpu_resources.cc:61] Please NOTE: device: 0, GPU Compute Capability: 8.6, Driver API Version: 11.8, Runtime API Version: 11.7
W0329 16:06:06.434868 18376 gpu_resources.cc:91] device: 0, cuDNN Version: 8.9.
[2024/03/29 16:06:07] ppocr INFO: train dataloader has 100 iters
[2024/03/29 16:06:07] ppocr INFO: valid dataloader has 100 iters
[2024/03/29 16:06:07] ppocr INFO: load pretrain successful from ./pretrain_models/ResNet18_vd_pretrained
now_debugg
[2024/03/29 16:06:07] ppocr INFO: During the training process, after the 0th iteration, an evaluation is run every 20 iterations

我的数据集一共100张图像和标注，望大佬解决！

**Comment by zhangyubo0722** [comment]
后台gpu有利用率吗

**Comment by kikiyaYang** [comment, marked as answer]
更新了版本，解决了

**Comment by zyyyyy123lao** [comment]
我也遇到了同样的问题，请问是更新了什么版本
